src/governance/governance.cpp

```diff
@@ -499,21 +575,25 @@ const CGovernanceObject* CGovernanceManager::FindConstGovernanceObject(const uin
 
 CGovernanceObject* CGovernanceManager::FindGovernanceObject(const uint256& nHash)
 {
-    AssertLockHeld(cs);
+    AssertLockNotHeld(cs);
+    LOCK(cs);
+    return InternalFindGovernanceObject(nHash);
+}
 
+CGovernanceObject* CGovernanceManager::InternalFindGovernanceObject(const uint256& nHash)
```

Comment:
The naming for internal methods is usually like that in our codebase
```suggestion
CGovernanceObject* CGovernanceManager::FindGovernanceObjectInternal(const uint256& nHash)
```
Pls consider renaming it

Same for other new `...Internal()`s.